pwiz_tools/Skyline/Controls/WizardPages.cs

```diff
@@ -50,6 +50,17 @@ protected override bool ProcessCmdKey(ref Message msg, Keys keyData)
             return base.ProcessCmdKey(ref msg, keyData);
         }
     }
+
+    public class WizardPageControl : UserControl

```

Comment:
Time will tell if this is better than an interface, since you currently only use this in one case and assume that not-implemented and base class have the same meaning of "true" the wizard can be closed. An interface would not need a base implementation and the single implementation would not need to call base.CanWizardClose().